.clang-format

```diff
@@ -5,6 +5,11 @@ IndentWidth: 4
 ContinuationIndentWidth: 8
 ColumnLimit: 100
 AllowShortFunctionsOnASingleLine: Empty
+
+WhitespaceSensitiveMacros:
+  - BETTER_ENUM
+  - INSTANTIATE_TEST_SUITE_P
```

Comment:
Isn't there a way to specify specific formatting for some macros instead of ignoring them completely?